WMD-group/SMACT
# New license

# License change

## Reviewers

I have added all contributors as reviewers - you just need to state that you are happy to have the license change. Feel free to dig deeper into the changes, but a record of your agreement to the change is the most important thing! Thanks.

As agreed with the developers of `smact` this PR changes from GPL to MIT license. 
* The LICENSE file has been changed
* GPL headers have been removed from all code files (no replacement is required for MIT)
* Reference to the `elements.txt` file from `OpenBabel` has been removed as we no longer use this file. The GPL 2 license associated with this has been removed.

## To test

No functional testing required. Please just check that license files have been changed where you would expect. Also, please confirm what I said about `OpenBabel` and GPL2.
